frontend/src/components/Inputs/PasswordInput.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+import eye from "../../assets/showPassword.svg";
+import "./Inputs.modules.scss";
+const PasswordInput = (props: {
+  name: any;
+  value: any;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  placeholder?: any;
```

Comment:
suggestion: avoid using `any` whenever possible.
I think the type of those props should be `string`